.release-please-manifest.json

```diff
@@ -0,0 +1,3 @@
+{
+    "tket2-py": "0.0.0"
```

Comment:
Is there any issue with having version 0.0.0? I seem to remember that this is somehow treated specially by some tools and it's better to use 0.1.0.

Reply:
This is just the "previous released version" for release-please to compute the relevant commits. Since there's no `tket-py-v0.0.0` tag it will just analyze all the history.

https://github.com/googleapis/release-please/blob/main/docs/manifest-releaser.md#initial-version